DisplaySettings/DisplaySettings.cpp

```diff
@@ -75,6 +75,7 @@ using namespace std;
 
 static bool isCecArcRoutingThreadEnabled = false;
 static bool isCecEnabled = false;
+static std::thread audioPortInitThread;
```

Comment:
Does this need to be static. Can we make it a member (of type Utils::ThreadRAII)?

Reply:
im not using threadRAII as it not serve the desire purpose. i can make audioPortInitThread to meember of displaySetting hhowever i choose this option as i want to use that in callback function which is static function so I have to pass as a data to thread funtion.